features/dd-sdk-android-logs/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt

```diff
@@ -44,8 +45,13 @@ internal class DatadogLogHandler(
         }
 
         val resolvedTimeStamp = timestamp ?: System.currentTimeMillis()
+        var combinedAttributes = attributes
+        val logsFeature = sdkCore.getFeature(Feature.LOGS_FEATURE_NAME)
+        if (logsFeature != null) {
+            combinedAttributes = logsFeature.unwrap<LogsFeature>().getAttributes().toMutableMap()
+            combinedAttributes.putAll(attributes)
```

Comment:
actually here we can simplify a bit, and considering my comment above, do something like:

```
val combinedAttributes = mutableMapOf<String, Any?>
val logsFeature = sdkCore.getFeature(Feature.LOGS_FEATURE_NAME)
if (logsFeature != null) {
  combinedAttributes.putAll(logsFeature.unwrap<LogsFeature>().getAttributes())
}
combinedAttributes.putAll(attributes)
```

Looks a bit cleaner and doesn't use `var`.

Reply:
Agreed. I'll make the change.